doc/specs/#1571 - New Tab Menu Customization/#1571 - New Tab Menu Customization.md

```diff
@@ -289,7 +399,39 @@ And assuming the user has bound:
     - Close Tab: `{ "action": "closeTab", "index": "${selectedTab.index}" }`
     - Close Other Tabs: `{ "action": "closeTabs", "otherThan": "${selectedTab.index}" }`
     - Close Tabs to the Right: `{ "action": "closeTabs", "after": "${selectedTab.index}" }`
-
+* We may want to consider regex, tag-based, or some other type of matching for
+  `matchProfile` entries in the future. We originally considered using regex for
+  `matchProfile` by default, but decided instead on full string matches to leave
+  room for regex matching in the future. Should we chose to pursue something
+  like that, we should use a settings structure like:
+
+  ```json
+  "type": "profileMatch",
+  "source": { "type": "regex", "value": ".*wsl.*" }
+  ```
+* We may want to expand `matchProfile` to match on other properties too. (`title`?)
+* We may want to consider something like
+  ```json
+  {
+    "type": "profileMatch",
+    "name": { "type": "regex", "value": "ssh: (.*)" }

```

Comment:
```suggestion
    "name": { "type": "regex", "value": "^ssh: (.*)" }
```

nit; Regex: anchor at the beginning. The more correct our samples, the more concise our thinking about how to deal w/ it :)